JsonLd/ResourceInterface.php

```diff
@@ -64,6 +64,20 @@ public function initFilters(array $filters);
     public function getFilters();
 
     /**
+     * Initializes order filters.
+     *
+     * @param array $orders
+     */
+    public function initOrder(array $orders);
```

Comment:
I would let `Order` since it's `order filters`. I will rename the `resourceOrders` `resourceOrder` and in the foreach `resourceOrder` by `orderElem`.